content/influxdb/cloud/write-data/troubleshoot.md

```diff
@@ -18,45 +18,48 @@ related:
 
 Learn how to handle and recover from errors when writing to InfluxDB.
 
-- [Discover common failure scenarios](#common-failure-scenarios)
+- [Discover common failure scenarios](#discover-common-failure-scenarios)
 - [Review HTTP status codes](#review-http-status-codes)
 - [Troubleshoot failures](#troubleshoot-failures)
 
-## Common failure scenarios
+## Discover common failure scenarios
 
 InfluxDB write requests may fail for a number of reasons.
 Common failure scenarios that return an HTTP `4xx` or `5xx` error status code include the following:
 
 - Exceeded a rate limit.
 - API token was invalid.
 - Client or server reached a timeout threshold.
-- Size of the data payload was too large.
 - Data was not formatted correctly.
 - Data did not conform to the [explicit bucket schema](/influxdb/cloud/organizations/buckets/bucket-schema/).
   See how to troubleshoot specific [bucket schema errors](/influxdb/cloud/organizations/buckets/bucket-schema/#troubleshoot-errors).
```

Comment:
we have the "troubleshoot failures" section describing what to do for rejected points---may be helpful to indicate what to do next for invalid API token, and lines 30-33? maybe 30-34 (if we didn't already provide suggestions for resolving non-conforming explicit bucket schema in referenced doc?)